### ISSUE TITLE
[Accessibility] Verfiy Checkbox Is Not Accessible

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/cas#365]
**[Original report](https://bitbucket.org/laborstudents/cas-flask/issue/365) by Hila Manalai (Bitbucket: [Hila Manalai](https://bitbucket.org/Hila Manalai), ).**

----------------------------------------

* **URL:** /courses/TERM\_CODE/COURSE

    ‌         For instance: 

    ‌         /courses/202012/MUS  

* **Task Attempted:** When a course is cross-listed with another course, a checkbox named Verify appears in the Course table. Checking the Verify checkbox means that the cross-listed course will be verified. However, there is no indication of such a change. So the user does not know if any changes have happened at all.
* **Expected Outcome:** Whenever the checkbox is checked there should be a flash message telling the user that the cross-listed course has been verified. And when it is unchecked it should say the cross-listed course is not verified. Something along those lines.

‌
